OpenRA.Game/Game.cs

```diff
@@ -368,7 +368,9 @@ public static void InitializeMod(string mod, Arguments args)
 			Sound.StopVideo();
 
 			ModData = new ModData(Mods[mod], Mods, true);
-			ExternalMods.Register(ModData.Manifest);
+			var manifest = ModData.Manifest;
+			if (!ExternalMods.ContainsKey(ExternalMod.MakeKey(manifest)))
```

Comment:
We can't unconditionally skip registration here.  If the launch path has changed (e.g. somebody has moved the .app bundle to a new location on macOS) then we need to update the metadata.